reactive/docker.py

```diff
@@ -152,5 +153,10 @@ def recycle_daemon():
 
 def reload_system_daemons():
     ''' Reload the system daemons from on-disk configuration changes '''
-    command = ['systemctl', 'daemon-reload']
-    check_call(command)
+    lsb = lsb_release()
+    code = lsb['DISTRIB_CODENAME']
+    if code != 'trusty':
+        command = ['systemctl', 'daemon-reload']
+        check_call(command)
+    else:
+        service_reload('docker')
```

Comment:
I have a strong feeling that running `service docker reload` would work on systemd computers as well, removing the need for this if statement. But as I don't have time to test this theory I will allow it. We should determine if reload works on xenial and come back to this in the future.